.buildkite/commands/build-for-testing.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+echo "--- :rubygems: Setting up Gems"
+restore_cache "$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"
```

Comment:
It might be worth extracting `"$(hash_file .ruby-version)-$(hash_file Gemfile.lock)"` into a variable to future-proof against changes not reflecting in subsequent usage(s).

Also, I find it interesting that `.ruby-version` has to be hashed as well as I expected `Gemfile.lock` to contain this information. It looks like it might be possible to do this with: https://bundler.io/gemfile_ruby.html?